src/custom/state/orders/utils.ts

```diff
@@ -227,14 +258,19 @@ export function getEstimatedExecutionPrice(
     return limitPrice
   }
 
-  const amountMinusFees = inputAmount.subtract(feeAmount)
+  // Check what's left to sell, discounting the surplus, if any
```

Comment:
ubermeganit preference on comment styling: Compared to the previous comments, I like this one because it's commanding something to the code (use the imperative). The other comments above don't